site/src/sections/About/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { AboutStyleWrapper } from './about.style'
+
+const Mainabout = () => {
+  return (
+    <AboutStyleWrapper>
+            <h1 className="heading">About the <span className='green'>Service Mesh Patterns</span> Book</h1>
```

Comment:
Change the indentation to 2 spaces for the whole component definition.

Reply:
done